docs/usage.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Usage hints for DietPi
+title: Usage hints / HowTo
+---
+
+# Usage hints / HowTo
+
+This chapter contains several documents describing the usage of DietPi.
+
+## How to use the logging mechanism
+
+DietPi uses the systemd based logging (instead of using syslog). `journald` is the logging part.
+The basic command to evaluate the systemd-journal is
```

Comment:
`syslog` is more a generic term for all kinds of system logging, so `systemd-journald` is a syslog daemon, like rsyslog, and one that is automatically present and necessarily used when `systemd` is used as init system. Probably we can make this clear:
```
DietPi uses systemd as system and service manager, which includes the `systemd-journald` logging daemon. An additional syslog daemon, like `rsyslog`, is not required and hence not pre-installed on DietPi. The basic command to access `systemd-journald` logs is
```

Reply:
Ok. Taken over.